packages/suite/src/support/extraDependencies.ts

```diff
@@ -1,25 +1,30 @@
 import { ExtraDependencies } from '@suite-common/redux-utils';
 import { NetworkSymbol } from '@suite-common/wallet-config';
 
-import { STORAGE } from '../actions/suite/constants';
+import { STORAGE, METADATA } from '../actions/suite/constants';
 import { addEvent } from '../actions/suite/notificationActions';
 import { StorageLoadAction } from '../actions/suite/storageActions';
-import { fetchAndUpdateAccount } from '../actions/wallet/accountActions';
 import type { BlockchainState } from '../reducers/wallet/blockchainReducer';
 import { AppState } from '../types/suite';
+import * as transactionActions from '@wallet-actions/transactionActions';
 
 export const extraDependencies: ExtraDependencies = {
-    thunks: { fetchAndUpdateAccount, notificationsAddEvent: addEvent },
+    thunks: { notificationsAddEvent: addEvent },
     selectors: {
         selectFeeInfo: (networkSymbol: NetworkSymbol) => (state: AppState) =>
             state.wallet.fees[networkSymbol],
-        selectAccounts: (state: AppState) => state.wallet.accounts,
         selectDevices: (state: AppState) => state.devices,
         selectBitcoinAmountUnit: (state: AppState) => state.wallet.settings.bitcoinAmountUnit,
     },
-    actions: {},
+    actions: {
+        // FIXME: these actions are temporary - remove it when Bohdan is ready with wallet-transactions!
+        addTransaction: transactionActions.add,
+        removeTransaction: transactionActions.remove,
+    },
     actionTypes: {
         storageLoad: STORAGE.LOAD,
+        metadataAccountLoaded: METADATA.ACCOUNT_LOADED,
```

Comment:
please add also reducers for this action types or better migrate them to `createAction` here and use `extra.actions` so you can use matchers in reducer builder and they will be properly typed. Now the payload is typed as any in reducer